pokemongo_bot/cell_workers/drop_items_worker.py

```diff
@@ -0,0 +1,134 @@
+                items_to_remove.append([Item.ITEM_HYPER_POTION.value, how_many, '{} Hyper Potion'.format(how_many)])
+                self.items[Item.ITEM_HYPER_POTION.value] = self.items[Item.ITEM_HYPER_POTION.value] - (how_many)
+                if self.items[Item.ITEM_SUPER_POTION.value]>20:
+                    how_many = self.items[Item.ITEM_SUPER_POTION.value] - 20
+                    items_to_remove.append([Item.ITEM_SUPER_POTION.value, how_many, '{} Super Potion'.format(how_many)])
+                    self.items[Item.ITEM_SUPER_POTION.value] = self.items[Item.ITEM_SUPER_POTION.value] - (how_many)
+                if self.items[Item.ITEM_POTION.value]:
+                    how_many = self.items[Item.ITEM_POTION.value]
+                    items_to_remove.append([Item.ITEM_POTION.value, how_many, '{} Potion'.format(how_many)])
+                    self.items[Item.ITEM_POTION.value] = 0
+            if self.items[Item.ITEM_SUPER_POTION.value] > 25:
+                how_many = self.items[Item.ITEM_SUPER_POTION.value] - 25
+                items_to_remove.append([Item.ITEM_SUPER_POTION.value, how_many, '{} Super Potion'.format(how_many)])
+                self.items[Item.ITEM_SUPER_POTION.value] = self.items[Item.ITEM_SUPER_POTION.value] - (how_many)
+                if self.items[Item.ITEM_POTION.value]>20:
+                    how_many = self.items[Item.ITEM_POTION.value] - 20
+                    items_to_remove.append([Item.ITEM_POTION.value, how_many, '{} Potion'.format(how_many)])
+                    self.items[Item.ITEM_POTION.value] = self.items[Item.ITEM_POTION.value] - (how_many)
+            if self.items[Item.ITEM_POTION.value] > 25:
+                how_many = self.items[Item.ITEM_POTION.value] - 25
+                items_to_remove.append([Item.ITEM_POTION.value, how_many, '{} Potion'.format(how_many)])
+                self.items[Item.ITEM_POTION.value] = self.items[Item.ITEM_POTION.value] - (how_many)
+
+            if self.items[Item.ITEM_MAX_REVIVE.value] > 25:
+                how_many = self.items[Item.ITEM_MAX_REVIVE.value] - 25
+                items_to_remove.append([Item.ITEM_MAX_REVIVE.value, how_many, '{} Max Revive'.format(how_many)])
+                self.items[Item.ITEM_MAX_REVIVE.value] = self.items[Item.ITEM_MAX_REVIVE.value] - (how_many)
+                if self.items[Item.ITEM_REVIVE.value]>20:
+                    how_many = self.items[Item.ITEM_REVIVE.value] - 20
+                    items_to_remove.append([Item.ITEM_REVIVE.value, how_many, '{} Revive'.format(how_many)])
+                    self.items[Item.ITEM_REVIVE.value] = self.items[Item.ITEM_REVIVE.value] - (how_many)
+            if self.items[Item.ITEM_REVIVE.value] > 25:
+                how_many = self.items[Item.ITEM_REVIVE.value] - 25
+                items_to_remove.append([Item.ITEM_REVIVE.value, how_many, '{} Revive'.format(how_many)])
+                self.items[Item.ITEM_REVIVE.value] = self.items[Item.ITEM_REVIVE.value] - (how_many)
```

Comment:
This GIGANTIC block of IFs makes me wanna cry. Why are you not using the `item_filter`? It's easy to add support to item names there, instead of IDs to make it more simple to the user.